src/d3q27_cumulant/Dynamics.c.Rt

```diff
@@ -121,17 +166,38 @@ CudaDeviceFunction void SetEquilibrum(real_t rho, real_t Jx, real_t Jy, real_t J
 }
 
 CudaDeviceFunction void Init() {
+	real_t k_aa = exp(-1/constContainer.ST.TimeWN);
+        real_t k_bb =  sqrt(1 - k_aa*k_aa);
 	vector_t ST = SyntheticTurbulence(0);
-	ST.x = Velocity + Turbulence*ST.x;
-	ST.y = Turbulence*ST.y;
-	ST.z = Turbulence*ST.z;
+	ST.x = Turbulence*ST.x*k_bb + SynthTX*k_aa;
+	ST.y = Turbulence*ST.y*k_bb + SynthTY*k_aa;
+	ST.z = Turbulence*ST.z*k_bb + SynthTZ*k_aa; 
+	SynthTX = ST.x;
+	SynthTY = ST.y;
+	SynthTZ = ST.z;
+	ST.x = ST.x + Velocity;	
 	SetEquilibrum(1.0 + Pressure * 3.0, ST.x, ST.y, ST.z);
 }
 
+CudaDeviceFunction void SynthInit() {
+        real_t k_aa = exp(-1/constContainer.ST.TimeWN);
+        real_t k_bb =  sqrt(1 - k_aa*k_aa);
+       	vector_t ST = SyntheticTurbulence(0);
```

Comment:
ST won't be visible in the boundary condition implementation, because it's local to SynthInit.